lighthouse-core/test/lib/drivers/driver-test.js

```diff
@@ -80,4 +82,23 @@ describe('Browser Driver', () => {
     assert.notEqual(opts.url, req1.url, 'opts.url changed after the redirects');
     assert.equal(opts.url, req3.url, 'opts.url matches the last redirect');
   });
+
+  it('will clear origin of a page', () => {
```

Comment:
This test doesn't enter into the driver code at all, just stays in this file. 

----

One possible way to "test" this is to instantiate a real driver, but then override sendCommand. We could than call `driver.clearDataForOrigin` and we'd at least be able to determine that we parsed an origin correctly

but that's really all we'd be testing. IMO not worth it, so we should just nuke the test, yeah.